src/share/classes/com/sun/javatest/regtest/exec/RegressionScript.java

```diff
@@ -1342,5 +1346,10 @@ private static String getVersion() {
     private Modules modules;
     private ScratchDirectory scratchDirectory;
     Locations locations;
+    private final TestProperties properties;
+
+    public RegressionScript(TestProperties properties) {
```

Comment:
I did not notice this in the previous round, sorry, but this constructor can be `private` like the other one that previously was.

Reply:
No, it has to remain `public` because it is used by `RegressionTestSuite` in the `config` package.

Reply:
You are right, I missed that.